src/ZombieDriver.php

```diff
@@ -32,6 +32,12 @@ class ZombieDriver extends CoreDriver
      */
     public function __construct($serverOrHost, $port = null)
     {
+        // Ensure that the "stop" method gets called no matter
+        // how the PHP execution finishes.
+        if (function_exists("register_shutdown_function")) {
+          register_shutdown_function(array($this, 'stop'));
```

Comment:
There is no need to. The Driver makes sure that there is a server before stopping it and the Server checks if the server is running before stopping it. I don't think there is a need for another layer of redundancy.